### PR TITLE
Andoni, Sarah, Justin added to About Us 

### PR DESCRIPTION
**Pull request by rowa97**

### CODE REVIEW

File: netlify/pages/about.yaml [hunk 1]
```diff
@@ -25,6 +28,9 @@ team:
   - image: /images/faust_headshot.png
     name: Scott Faust
     role: Growth & Ops
+  - image: /images/ash-edmonds-636863-unsplash.jpg
+    name: 'Justin Forte '
```
Comment (tbarn) [inline netlify/pages/about.yaml]: Remove space after name

File: netlify/pages/about.yaml [hunk 2]
```diff
@@ -25,6 +28,9 @@ team:
   - image: /images/faust_headshot.png
     name: Scott Faust
     role: Growth & Ops
+  - image: /images/ash-edmonds-636863-unsplash.jpg
```
Comment (tbarn) [inline netlify/pages/about.yaml]: Did you mean to not have an actual photo here?

Comment (rowa97) [reply netlify/pages/about.yaml]: nope, still waiting on Justin to send me one

Comment (lottamus) [reply netlify/pages/about.yaml]: @rowa97 can you remove him for now until we have a picture? That way we can go ahead and get this release out that has the pricing changes 🤓

Comment (tbarn) [reply netlify/pages/about.yaml]: I'll fix this.

Comment (tbarn) [reply netlify/pages/about.yaml]: @lottamus I lied, I can't because I can't edit develop without a PR. Can you do it?

Comment (rowa97) [reply netlify/pages/about.yaml]: fixed

**Pull request merged**